Sprint-1/exercises/random.js

```diff
@@ -1,7 +1,15 @@
 const minimum = 1;
 const maximum = 100;
 
-const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
+// Calculate how many numbers we can pick
+const valueRange = maximum - minimum + 1;
+// Get a random number and make it fit our range
+const scaledRandomNumber = Math.random() * (valueRange);
+// Round the random number down to the nearest whole number
+const roundedDownValue = Math.floor(scaledRandomNumber);
+//Add the smallest number to the rounded number to get the final number
+const num = roundedDownValue + minimum;
+console.log(num);
```

Comment:
To test your understanding, how would you write an expression (without using any variable in the expression) that yields a random integer between -5 and 3?

Reply:
`Math.floor ( Math.random( ) * (3 - (-5) + 1) ) + (-5);`